extensions/amp-story/0.1/amp-story-cta-layer.js

```diff
@@ -43,6 +43,23 @@ export class AmpStoryCtaLayer extends AmpStoryBaseLayer {
      */
     return false;
   }
+
+  /** @override */
+  buildCallback() {
+    super.buildCallback();
+    this.setOrOverwriteTargetAttribute_();
+  }
+
+  /**
+   * Overwrite or set target attribute to _blank in call-to-action links.
+   * @private
+   */
+  setOrOverwriteTargetAttribute_(){
+    const ctaLinks = this.element.querySelectorAll('a');
+    for (let i = 0; i < ctaLinks.length; i++) {
+      ctaLinks[i].target = '_blank';
```

Comment:
Nit: I have no idea if it makes a difference, but the rest of the codebase uses ``element.setAttribute('target', '_blank')``. Maybe we should write it this way too?